docs/content/concepts/automation/declarative-automation.mdx

```diff
@@ -133,6 +143,24 @@ def my_eager_asset(): ...
 AssetSpec("my_cron_asset", automation_condition=AutomationCondition.on_cron("@daily"))
 ```
 
+The same is true for asset checks:
+
+```python
+from dagster import AssetCheckResult, AssetCheckSpec, AutomationCondition, asset_check
+
+
+@asset_check(asset=..., automation_condition=AutomationCondition.cron_tick_passed("@daily"))
```

Comment:
too late for this feedback probably but something about `cron_tick_passed` as a condition feels off. The notion of a cron tick is so specific and internal to Dagster.

Reply:
Curious to hear more about this bit -- in my mind a "cron tick" is more of a general concept, i.e. whenever you define a cron schedule, that schedule is just an infinite series of ticks. Not sure if there's a more industry standard term than "tick" for "distinct time at which a cron schedule would kick off"